src/GlpiProject/API/Rest/Client.php

```diff
@@ -1,9 +1,41 @@
 <?php
+/**
+ * LICENSE
+ *
+ * Copyright © 2017 Teclib'
+ *
+ * This file is part of GLPI Api Client Library for PHP.
+ *
+ * GLPI Api Client Library for PHP is a subproject of Flyve MDM. Flyve MDM is a mobile
+ * device management software.
+ *
+ * GLPI Api Client Library for PHP is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * GLPI Api Client Library for PHP is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with GLPI Api Client Library for PHP. If not, see http://www.gnu.org/licenses/.
+ * ------------------------------------------------------------------------------
+ * @author    Domingo Oropeza
+ * @copyright Copyright © 2017 Teclib
+ * @license   AGPLv3+ http://www.gnu.org/licenses/agpl.txt
+ * @link      https://github.com/flyve-mdm/composer-package-glpi
+ * @link      https://flyve-mdm.com/
+ * ------------------------------------------------------------------------------
+ */
+
 namespace GlpiProject\API\Rest;
```

Comment:
I prefer this namespace: `GLPI\API\REST;`

Reply:
That means that the directories has to change because the PRS recommendations. 

This could be done but I like more this namespace `Glpi/ApiClient`. What do you think about this @btry ?

Reply:
Hi, @DIOHz0r 

In UPPERCASE because there are acronyms and abbreviations.

GLPI\API\REST (JSON)
GLPI\API\RPC (XML)

Reply:
Hi

These words are all acronyms, but I think we should not abuse uppercase. (the backslashes are lost in this mess).

The already existing build tools use the following namespace: ```Glpi\Tools```. This is already established, and we shall follow it. 

So I vote for one of the following proposals:
* ```Glpi\API\REST``` 
* ```Glpi\Api\Rest``` 

The second one is more consistent as Glpi is CamelCase despite it is an acronym, so other acronyms should probably use the same rule.